scripts/patch/build.diff

```diff
@@ -1,8 +1,8 @@
 diff --git a/config/compiler/BUILD.gn b/config/compiler/BUILD.gn
```

Comment:
Out of curiosity, how are we generating these patches?

Reply:
Manually :(

If we clean up some of this, perhaps we can push a PR upstream to Google.